jwst/cube_build/ifu_cube.py

```diff
@@ -1261,9 +1260,12 @@ def setup_ifucube_wcs(self):
                     if ch1 in self.list_par1 or ch3 in self.list_par1:
                         spatial_found = False
                         spectral_found = False
-
+                # If Moving TARGET data then do not use s_region the y are not updated for moving targets
+                target_type = input_model.meta.target.type
+                if target_type == 'MOVING':
+                    spatial_found = False
                 if(spectral_found & spatial_found & world):
```

Comment:
I'm guessing the beefed up flake8 checking is going to want a whitespace after the "if" in this line (there really should be anyway, just for readability).